src/Test/Inspection/Plugin.hs

```diff
@@ -22,21 +22,27 @@ import Outputable
 import Test.Inspection (Obligation(..), Property(..), Result(..))
 import Test.Inspection.Core
 
--- | The plugin. It supports the option @-fplugin-opt=Test.Inspection.Plugin:keep-going@ to
--- ignore a failing build.
+-- | The plugin. It supports some options:
+--
+-- * @-fplugin-opt=Test.Inspection.Plugin:keep-going@ to ignore a failing build
+-- * @-fplugin-opt=Test.Inspection.Plugin:quiet@ to not print fulfilled obligations
```

Comment:
better “to be silent if all obligations are fulfilled” – as the flag also disable the summary line if all obligations are fulfilled.